app/src/main/res/values/strings.xml

```diff
@@ -10,4 +10,9 @@
     <string name="registerFailed">Registriation failed</string>
     <string name="cardImageDescription">Wizard card</string>
     <string name="gameActivity">Game Activity</string>
+    <string name="showPoints">Show Points</string>
+    <string name="predictTrickNumber">Predict the number of tricks for this round!</string>
+    <string name="predictionScore">Highest score with your Prediction:</string>
+    <string name="confirm">Confirm</string>
+    <string name="roundPlaceholder">Round: 1 of 15</string>
```

Comment:
I think we should use a prefix for all Strings used in a specific context

Reply:
We can, but it would probably be better to make a separate refactoring PR where we do that for all string and dimension resources

Reply:
Sounds good just create a issue for that

Reply:
Created #33 for this